controllers/transactionsController.js

```diff
@@ -0,0 +1,25 @@
+const transactionsModel = require('../models/transaction')
+
+/**
+ * Collects all transactions and sends only required data for each userId
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const fetchLatestTransactions = async (req, res) => {
+  try {
+    const userId = req.params.userId
+    const data = await transactionsModel.fetchLatestTransactions(userId)
```

Comment:
We should be sending `username` in the URL. The userId is a random string, we shouldn't be using that to make the calls. 

We should be getting the `username` from the URL and then find the `userId` for that `username`. We already have functions which do that, please check: `utils/users.js`

Reply:
Please check this comment @rkmr039